fw/application/src/app/amiidb/scene/amiidb_scene_amiibo_search.c

```diff
@@ -23,6 +23,21 @@ static void amiidb_scene_amiibo_search_text_input_event_cb(mui_text_input_event_
     }
 }
 
+static void amiidb_msg_box_error_cb(mui_msg_box_event_t event, mui_msg_box_t *p_msg_box) {
+    app_amiidb_t *app = p_msg_box->user_data;
+    mui_scene_dispatcher_next_scene(app->p_scene_dispatcher, AMIIDB_SCENE_AMIIBO_SEARCH);
```

Comment:
It's better to call mui_view_dispatcher_switch_to_view(app->p_view_dispatcher, AMIIDB_VIEW_ID_LIST);
because there isn't scene switch over, just switching the view should be fine.